src/form/Input.js

```diff
@@ -50,142 +47,181 @@ const placeholder = field => {
   <FormGroup className={className || field.className}>
-    <InputLabel field={field}/>
-    {field.choices && field.choices.map(prep_choice).map((c, i) => (
-      <div key={i} className="custom-control custom-radio">
-        <input type="radio"
-               className="custom-control-input"
-               checked={c.value === value}
-               disabled={disabled}
-               name={field.name}
-               id={`${field.name}-${c.value}`}
-               required={field.required}
-               onChange={() => onChange(c.value)}
-               onBlur={() => onBlur(c.value)}/>
-        <label className="custom-control-label" htmlFor={`${field.name}-${c.value}`}>
-          <span className="d-inline-block mr-1">{c.label.title || c.label}</span>
-          {c.label.description && <small className="text-muted">{c.label.description}</small>}
-        </label>
-     </div>
-    ))}
+    <InputLabel field={field} />
+    {field.choices &&
+      field.choices.map(prep_choice).map((c, i) => (
+        <div key={i} className="custom-control custom-radio">
+          <input
+            type="radio"
+            className="custom-control-input"
+            checked={c.value === value}
+            disabled={disabled}
+            name={field.name}
+            id={`${field.name}-${c.value}`}
+            required={field.required}
+            onChange={() => onChange(c.value)}
+            onBlur={() => onBlur(c.value)}
+          />
+          <label className="custom-control-label" htmlFor={`${field.name}-${c.value}`}>
+            <span className="d-inline-block mr-1">{c.label.title || c.label}</span>
+            {c.label.description && <small className="text-muted">{c.label.description}</small>}
+          </label>
+        </div>
+      ))}
     {error && <FormFeedback className="d-block">{error}</FormFeedback>}
-    <InputHelpText field={field}/>
+    <InputHelpText field={field} />
   </FormGroup>
 )
 
-export const InputToggle = ({className, field, disabled, error, value, onChange, onBlur}) => (
+export const InputToggle = ({className, field, disabled, error, value, onChange, onBlur, extraButton}) => (
   <FormGroup className={className || field.className}>
-    <InputLabel field={field}/>
+    <InputLabel field={field} />
     <div>
       <ButtonGroup>
-        {field.choices && field.choices.map(prep_choice).map((c, i) => (
-          <Button key={i} disabled={disabled} onClick={() => onChange(c.value)}
-                  color={c.value === value ? 'primary' : 'secondary'}>
-            {c.label}
+        {field.choices &&
+          field.choices.map(prep_choice).map((c, i) => (
+            <Button
+              key={i}
+              disabled={disabled}
+              onClick={() => onChange(c.value)}
+              color={c.value === value ? 'primary' : 'secondary'}
+            >
+              {c.label}
+            </Button>
+          ))}
+        {extraButton && (
```

Comment:
how are you using `extraButton` in nell?

Would it be more flexible to use `children` do you can add whatever you like here?

Reply:
Yes I think you're right... I'll change it to use children rather